examples/compose/porter.yaml

```diff
@@ -17,24 +17,24 @@ mixins:
 
 customActions:
```

Comment:
Heads up, there isn't a need to declare a custom action unless you are setting `modifies: false` or `stateless: true`. Otherwise you can just use a custom action and porter will define it for you with `modifies: true` and `stateless: false`.

Reply:
Oh nice!  I missed that.  I like it.